pygmt/clib/session.py

```diff
@@ -682,7 +682,7 @@ def _check_dtype_and_dim(self, array, ndim):
 
         Parameters
         ----------
-        array : numpy array
+        array : numpy.ndarray
             The array to be tested.
         ndim : int
             The desired dimension of the array.
```

Comment:
This has to be changed similar above in the lines 680-681, please. (GitHub does not allow me to make a suggestion there).
```suggestion
        ndim : int
            The desired number of dimensions of the array.
```